recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -19,9 +19,10 @@ CCACHE_DISABLE[unexport] = "1"
 
 TOOLCHAIN = "gcc"
 
-PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg"
+PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg atk"
```

Comment:
I'm not sure this should be enabled by default.

Reply:
I don't mind either way :)

Reply:
> Hah. Funny, I had a similar commit locally, I had to fix poky upstream to remove their X11 requirement for at-spi2-core btw.

Nice. Now I understand why was working out-of-the-box today.

Reply:
> I don't mind either way :)

makes sense